commands.json

```diff
@@ -1517,7 +1517,7 @@
     "group": "list"
   },
   "LPUSHX": {
-    "summary": "Prepend a value to a list, only if the list exists",
+    "summary": "Prepend one or multiple values to a list, only if the list exists",
     "complexity": "O(1)",
```

Comment:
Should be "O(N) where N is the number of elements to be added by the operation"

Reply:
updated complexity and history in commit 9a6afdd6f9